blackbox/docs/installation.txt

```diff
@@ -108,3 +107,4 @@ additional packages if we see enough interest.
 .. _`docker-crate`: https://github.com/crate/docker-crate/
 .. _`Multi Node Setup`: https://crate.io/docs/scale/multi_node_setup/
 .. _`Java 8`: http://www.oracle.com/technetwork/java/javase/downloads/index.html
+.. _Hello CrateDB: https://crate.io/docs/crate/guide/tutorials/hello.html
```

Comment:
Multiple word links like "Hello CrateDB" have to be wrapped in backticks, I think